src/In.ProjectEKA.HipService/Discovery/PatientController.cs

```diff
@@ -27,4 +31,38 @@ public async Task<ActionResult> Discover(DiscoveryRequest request)
             return Ok(discoveryResponse);
         }
     }
+
+    [Route("patients/care-contexts/discover")]
+    [ApiController]
+    public class CareContextDiscoveryController : Controller
+    {
+        private readonly PatientDiscovery patientDiscovery;
+
+        public CareContextDiscoveryController(PatientDiscovery patientDiscovery)
+        {
+            this.patientDiscovery = patientDiscovery;
+        }
+
+        [HttpPost]
+        public AcceptedResult DiscoverPatientCareContexts(DiscoveryRequest request)
+        {
+            BackgroundJob.Enqueue(() => GetPatientCareContext(request));
+            return Accepted();
+        }
+
+        public Task GetPatientCareContext(DiscoveryRequest request)
+        {
+            var (discoveryResponse, errorRepresentation) =
+                patientDiscovery.PatientFor(request).GetAwaiter().GetResult();
+            var timestamp = DateTime.Now.ToUniversalTime().ToString(Constants.DateTimeFormat);
+            GatewayDiscoveryRepresentation response = new GatewayDiscoveryRepresentation(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'response'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5550197)

Reply:
Will make the gateway call with this response.